static/js/components/NewInstrument.jsx

```diff
@@ -226,6 +226,11 @@ const NewInstrument = () => {
         title:
           "Configuration data i.e. {'overhead_per_exposure': 2.0, 'readout': 8.0, 'slew_rate': 2.6, 'filt_change_time': 60.0}",
       },
+      treasuremap_id: {
+        type: "integer",
+        title: "Treasuremap ID",
+        description: "ID of the instrument to submit to Treasuremap (optional)",
```

Comment:
I guess we already have ZTF and DECam and a few others hardcoded somewhere. Can you put them by your FILTERS list for easy finding?

Reply:
You mean just to keep track of it? Or to try use that list when not added to the instrument in the DB?

Reply:
Yeah the latter is what I am thinking. I am not sure if we have done a good job mapping things, but I might be wrong.

Reply:
We havent mapped the ids anywhere. They are in the instruments model, and one can post it to the DB. We just have it added to ZTF in the demo data. But I can add a mapper for sure.

Reply:
Ok that might be good then. We want to be able to add it to the instruments model of course, but nothing wrong with a few common ones.

Reply:
I added the mapper with a bunch of instruments we have on Fritz, and some common ones that might be on other instances.

Reply:
Seems good. should be good to go once tested.